core/git-access.c

```diff
@@ -794,18 +821,16 @@ static struct git_repository *get_remote_repo(const char *localdir, const char *
 		}
 		return update_local_repo(localdir, remote, branch, rt);
 	} else {
-		/* we have no local cache yet */
-		if (is_subsurface_cloud) {
-			/* and take us temporarly online to create a local and
-			 * remote cloud repo.
-			 */
-			git_repository *ret;
-			bool glo = prefs.git_local_only;
-			prefs.git_local_only = false;
-			ret = create_local_repo(localdir, remote, branch, rt);
-			prefs.git_local_only = glo;
-			return ret;
-		}
+		/* We have no local cache yet.
+		 * Take us temporarly online to create a local and
+		 * remote cloud repo.
+		 */
+		git_repository *ret;
+		bool glo = prefs.git_local_only;
+		prefs.git_local_only = false;
```

Comment:
maybe `const bool glo` but that's just a nitpick and it may produce a warning.

Reply:
I'll leave this change to the original author (@dirkhh, I presume). The problem I see is that, if we start `const`ifying local variables, this will take no end...

Reply:
understood.